### DISCUSSION TITLE
session.strategy does not work

### DISCUSSION THREAD

**Discussion by s1monb** [nextauthjs/next-auth#3285] [Help] [answered]
I am using next-auth@4.0.0-next.26

According to the docs, this should work:
```
session: {
    strategy: 'jwt',
 },
```
Well, it does not seem to be working on my end, until I changed it to:
```
session: {
    jwt: true,
 },
```

I saw that this was the v3 way of doing according to the docs, but now I'm confused...

**Comment by balazsorban44** [comment, marked as answer]
Upgrade to `next-auth@beta`. `next.26` isn't maintained, and is considered an old/intermediary release.

FYI, v4 goes stable next week. :tada:
